packages/components/src/tabs/tablist.tsx

```diff
@@ -26,7 +27,8 @@ export const TabList = forwardRef<
 >( function TabList( { children, ...otherProps }, ref ) {
 	const context = useTabsContext();
 
-	const selectedId = context?.store.useState( 'selectedId' );
+	const tabStore = useStoreState( context?.store );
```

Comment:
Nit: the tab store is actually the one in `context.store`. This one is the store state

```suggestion
	const tabStoreState = useStoreState( context?.store );
```

Reply:
Right, thanks. Renamed as part of c9b238f3e3faf8f5e6ce711b79c73e10f9841a8f